src/service/real-time-config/callout-vendors.js

```diff
@@ -155,10 +155,10 @@ const RTC_VENDORS = jsonConfiguration({
     disableKeyAppend: true,
   },
   openwrap: {
-    // PubMatic OpenWrap
+    // PubMatic OpenWrap 
     url:
-      'https://ow.pubmatic.com/amp?v=1&w=ATTR(width)&h=ATTR(height)&ms=ATTR(data-multi-size)&auId=ATTR(data-slot)&purl=HREF&pubId=PUB_ID&profId=PROFILE_ID',
-    macros: ['PUB_ID', 'PROFILE_ID'],
+      'https://ow.pubmatic.com/amp?v=1&w=ATTR(width)&h=ATTR(height)&ms=ATTR(data-multi-size)&auId=ATTR(data-slot)&purl=HREF&pubId=PUB_ID&profId=PROFILE_ID&consent_string=CONSENT_STRING&gdpr_applies=CONSENT_METADATA(gdprApplies)&addtl_consent=CONSENT_METADATA(additionalConsent)&consent_type=CONSENT_METADATA(consentStringType)',
+    macros: ['PUB_ID', 'PROFILE_ID','CONSENT_STRING'],
```

Comment:
add space after `,` should fix your lint error

Reply:
Added an space.

Reply:
Thanks